### DISCUSSION TITLE
[Enhancement] [date-picker] How to make the date picker panel display all the time, can you add such a feature?

### DISCUSSION THREAD

**Discussion by stupidream** [element-plus/element-plus#18011] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

Could you add a feature that keeps the date panel displayed and won't be turned off, like the Calendar component?

### Additional comments

_(empty)_

**Comment by stupidream** [comment]
能否加一个让日期面板单独使用不会关闭的功能，就像Calendar组件的那种。

**Comment by Dsaquel** [comment]
Released in `2.11.0`.
https://element-plus.org/en-US/component/date-picker-panel.html
